_posts/2021-03-04-⚡-lighting-fast-pages.md

```diff
@@ -28,7 +28,7 @@ In addition to instrumenting ControlShift in the lab for front-end performance,
 Improving the page speed for those taking action via petitions and events has been a particular focus for us because these pages are seen by millions of supporters across our customer sites each month. Many of these supporters visit us with an empty browser cache, and they arrive primarily via mobile devices. We've implemented numerous new strategies to make these flows feel faster, especially on mobile devices:
 
 * Modernized the front-end framework we use to develop ControlShift and refined the CSS we generate to make it smaller. This was an enormous project for us but has paid significant dividends already. 
-* Switched from gzip compression for CSS and JavaScript assets to the newer and more efficient [brotli compression](https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html).
+* Switched from gzip compression for CSS and JavaScript assets to the newer and more efficient [Brotli compression](https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html).
```

Comment:
I decided this should be capitalized since it is in https://opensource.googleblog.com/2015/09/introducing-brotli-new-compression.html and wikipedia, even though it's not capitalized in https://www.ietf.org/rfc/rfc7932.txt ...  ¯\_(ツ)_/¯